qtglean/metrics.yaml

```diff
@@ -56,6 +56,28 @@ settings:
     expires: never
 
 session:
+  dns_type:
+    type: string
+    description: |
+      The type of DNS a user has active.
+      NoCustomDNSAvailable - Custom DNS app feature isn't available
+      Default - App's default DNS server
+      Custom - User has provided a valid IP address for their DNS
+      (BlockAds | BlockTrackers | BlockAdsAndTrackers
+      | BlockMalware | BlockMalwareAndAds | BlockMalwareAndTrackers
+      | BlockMalwareAndAdsAndTrackers) - Using app's privacy DNS option
```

Comment:
nit: You can use Markdown here. Maybe reformat it as a list.

Reply:
Thanks for that - I've updated this.